server/index.ts

```diff
@@ -51,31 +45,26 @@ if (!process.env.MONGODB_URI) {
 // GET all users
 app.get('/api/users/', async (_request, response) => {
   const usersCollection = getUserCollection();
-  const currywurst = usersCollection.find();
-  const allUsers = await currywurst.toArray();
+  const user = usersCollection.find();
+  const allUsers = await user.toArray();
   response.send(allUsers);
 });
```

Comment:
If you want to catch errors, see https://expressjs.com/en/guide/error-handling.html

```ts
app.get('/api/users', async (_request, response, next) => {
  try {
    const cursor = usersCollection.find();
    const allUsers = await user.toArray();
    response.send(allUsers);
  } catch(error) {
    next(error);
  }
});